package.json

```diff
@@ -54,17 +54,19 @@
     "xregexp": "3.2.0"
   },
   "devDependencies": {
-    "babel-cli": "6.26.0",
+    "babel-cli": "^6.26.0",
```

Comment:
Can we remove the `^` from all these? Don't want them auto-updating

Reply:
Sure